docs/concepts/gateway/customization.md

```diff
@@ -259,6 +259,44 @@ Jina provides the Gateway with a list of ports and protocols to expose. Therefor
 on multiple ports using different protocols.
 ```
 
+(executor-streamer)=
+## Calling an Individual Executor with {class}`~jina.serve.streamer.ExecutorStreamer`
```

Comment:
I would hide the class itself from the title, just ## Calling an Individual Executor